pytket/qir/conversion/conversion.py

```diff
@@ -747,13 +787,35 @@ def condition_block() -> None:
                     else:
                         get_gate(*qubits)
         if record_output:
-            for creg in self.circuit.c_registers:
-                reg_name = creg[0].reg_name
+
+            if len(self.reg_const) > 0:
```

Comment:
I don't think this condition should be necessary.

Reply:
It is required because we cant define the function `__quantum__rt__int_record_output` without the type of the constant.

Reply:
We could add an additional constant to avoid this if you want to?

Reply:
remove in @cqc-melf
[add point type](https://github.com/CQCL/pytket-qir/pull/38/commits/3439488874d80935be430393f16f0ac0aae1dc4a)
[3439488](https://github.com/CQCL/pytket-qir/pull/38/commits/3439488874d80935be430393f16f0ac0aae1dc4a)